Tuist/Interfaces/SwiftUI/Project/Podfile

```diff
@@ -12,9 +12,12 @@ end
 target '{PROJECT_NAME}' do
   # UI
   pod 'Kingfisher'
+  pod 'FlowStacks'
```

Comment:
Not every project will use FlowStacks so I think it is optional and not the default
```suggestion
```

Reply:
solved in [232caf4](https://github.com/nimblehq/ios-templates/pull/506/commits/232caf454645aac9cc6485e5a747b4e1f1ca0fd9)